src/main/java/mods/railcraft/common/items/firestone/ItemFirestone.java

```diff
@@ -65,4 +70,13 @@ public boolean isRepairable() {
         return false;
     }
 
+    @Override
+    public void finalizeDefinition() {
+        ItemStack ore = EnumOreMagic.FIRESTONE.getStack();
+        ItemStack rawfirestone = RailcraftItems.FIRESTONE_RAW.getStack();
```

Comment:
This class is used for both Raw and Cut. You'll probably to make a subclass for Raw and put this stuff there. Otherwise it will be defined twice.